mappings/net/minecraft/client/gui/screen/SplashOverlay.mapping

```diff
@@ -17,6 +19,10 @@ CLASS net/minecraft/class_425 net/minecraft/client/gui/screen/SplashOverlay
 		ARG 4 reloading
 	METHOD method_18103 renderProgressBar (Lnet/minecraft/class_4587;IIIIF)V
 		ARG 1 matrices
+		ARG 2 x1
+		ARG 3 y1
+		ARG 4 x2
+		ARG 5 y2
```

Comment:
Hmm, how about `min`/`max` (`start`/`end`) `X`/`Y`? Or if we stick with numbers, `01` or `12` suffixes?

Reply:
Looking at it without looking at the function made me realize that number based names are not very clear.
I like `min`/`max`, should it be `minX` or `xMin`?

Reply:
`minX` is the Yarn convention for that.